### PR TITLE
Add mummy_finance, opx_finance, minerva_money, fxdx, nex and avt to ```perpetual_trades```

### PR DESCRIPTION
**Pull request by kaiblade**
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

File: models/avt/optimism/avt_optimism_v1_perpetual_trades.sql [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+FROM {{ source('optimism','logs') }}
+
+WHERE contract_address = 0x24ee37267842a525c66fe37cd0da749150e89866
+  AND topic0 = 0x93d75d64d1f84fc6f430a64fc578bdd4c1e090e90ea2d51773e626d19de56d30
+  AND tx_hash IN ( SELECT evt_tx_hash FROM {{ source('avt_optimism', 'PositionRouter_evt_ExecuteDecreasePosition') }} )
+    {% if not is_incremental() %}
+  AND block_time >= DATE '{{project_start_date}}'
+    {% endif %}
+    {% if is_incremental() %}
+  AND block_time >= DATE_TRUNC('DAY', NOW() - INTERVAL '7' Day)
```
Comment (Hosuke) [inline models/avt/optimism/avt_optimism_v1_perpetual_trades.sql:77]: You can use `incremental_predicate` in future:
```suggestion
  AND {{ incremental_predicate('block_time') }}
```

File: models/fxdx/optimism/fxdx_optimism_v2_perpetual_trades.sql [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          {% endif %}
+          {% if is_incremental() %}
+          AND evt_block_time >= DATE_TRUNC('DAY', NOW() - INTERVAL '7' Day)
+          {% endif %}
+      INNER JOIN margin_fees_info fee
+          ON event.evt_tx_hash = fee.evt_tx_hash
+          AND event.evt_index > fee.evt_index
+          AND event.evt_index < fee.next_evt_index
+      INNER JOIN {{ ref('tokens_optimism_erc20') }} tokens
+          ON event.indexToken = tokens.contract_address
```
Comment (Hosuke) [inline models/fxdx/optimism/fxdx_optimism_v2_perpetual_trades.sql:93]: You can use `tokens_erc20` here with a filter for generic approach.
```suggestion
      INNER JOIN {{ ref('tokens_erc20') }} tokens
          ON event.indexToken = tokens.contract_address
          AND tokens.blockchain = 'optimism'
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @kaiblade 

[Sample avt_optimism_v1_perpetual_trades](https://dune.com/queries/3328317)
[Sample fxdx_optimism_v2_perpetual_trades](https://dune.com/queries/3328326)
[Sample minerva_money_optimism_v1_perpetual_trades](https://dune.com/queries/3328330)
[Sample mummy_finance_optimism_v1_perpetual_trades](https://dune.com/queries/3328335)
[Sample nex_optimism_perpetual_trades](https://dune.com/queries/3328338)
[Sample opx_finance_optimism_v1_perpetual_trades](https://dune.com/queries/3328341)

File: models/avt/optimism/avt_optimism_v1_perpetual_trades.sql [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  AND topic0 = 0x2fe68525253654c21998f35787a8d0f361905ef647c854092430ab65f2f15022
+  AND tx_hash IN ( SELECT evt_tx_hash FROM {{ source('avt_optimism', 'PositionRouter_evt_ExecuteIncreasePosition') }} )
+    {% if not is_incremental() %}
+  AND block_time >= DATE '{{project_start_date}}'
+    {% endif %}
+    {% if is_incremental() %}
+  AND block_time >= DATE_TRUNC('DAY', NOW() - INTERVAL '7' Day)
+    {% endif %}
+
+UNION
```
[the remark this answers is not in the document]
Comment (jeff-dude) [reply models/avt/optimism/avt_optimism_v1_perpetual_trades.sql:49]: better performance on dunesql, if possible to do `union all`

**Review by jeff-dude — CHANGES_REQUESTED** [review]
i only left feedback for `avt`, but i imagine it applies universally in the PR. please do go ahead and make suggested changes

File: models/avt/optimism/avt_optimism_v1_perpetual_trades.sql [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+             trx."from",
+             trx.to,
+             fees.feeUsd    AS margin_fee
+      FROM all_executed_positions event
+      INNER JOIN {{ source('optimism', 'transactions') }} trx
+      ON event.tx_hash = trx.hash
+          {% if not is_incremental() %}
+          AND event.block_time >= DATE '{{project_start_date}}'
+          {% endif %}
+          {% if is_incremental() %}
```
[the remark this answers is not in the document]
Comment (kaiblade) [reply models/avt/optimism/avt_optimism_v1_perpetual_trades.sql]: @jeff-dude Addressed all changes.

**Review by jeff-dude — APPROVED** [review]
looks good now 🤝

**Pull request merged**